INSTALL.md

```diff
@@ -3,98 +3,134 @@
-to build Singularity from source.
+To build a stable version of Singularity, check out a [release tag](https://github.com/sylabs/singularity/tags) before compiling:
 
 ```
-$ mkdir -p $GOPATH/src/github.com/sylabs
-$ cd $GOPATH/src/github.com/sylabs
-$ git clone https://github.com/sylabs/singularity.git
-$ cd singularity
+$ git checkout v3.0.3
 ```
 
-## Compile the Singularity binary
+## Compiling Singularity
+
 You can build Singularity using the following commands:
 
 ```
-$ cd $GOPATH/src/github.com/sylabs/singularity
-$ ./mconfig
-$ cd ./builddir
-$ make
-$ sudo make install
+$ cd ${GOPATH}/src/github.com/sylabs/singularity && \
+  ./mconfig && \
+  cd ./builddir && \
+  make && \
+  sudo make install
 ```
 
-Alternatively, to build an rpm on CentOS/RHEL use the following commands:
+And that's it! Now you can check your Singularity version by running:
 
 ```
-$ sudo yum install -y rpm-build
-$ cd $GOPATH/src/github.com/sylabs/singularity
-$ ./mconfig
-$ make -C builddir rpm
+$ singularity version
 ```
+To build in a different folder and to set the install prefix to a different path:
 
-To build an rpm with an alternative install prefix set RPMPREFIX on the
-make step, for example
 ```
-$ make -C builddir rpm RPMPREFIX=/usr/local
+$ ./mconfig -p /usr/local -b ./buildtree
 ```
 
-To build a stable version of Singularity, check out a [release tag](https://github.com/sylabs/singularity/tags) before compiling:
+## Install from the RPM
+
+*NOTE: You should only atempt to build the RPM on a CentOS/RHEL system.*
+
+To build the RPM, you first need to install `rpm-build` and `wget`.
 
 ```
-$ git checkout v3.0.3
+$ sudo yum -y update && sudo yum install -y rpm-build wget
 ```
 
-To build in a different folder and to set the install prefix to a different path:
+Then download the latest 
+[release tarball](https://github.com/sylabs/singularity/releases) and use it to
+install the RPM.  Since we are building from the RPM, you don't need to install 
+Golang, but you do need to 
+[install the other dependencies](#install-system-dependencies).
 
 ```
-$ ./mconfig -p /usr/local -b ./buildtree
+$ export VERSION=3.0.3  # this is the singularity version, change as you need
+
+$ wget https://github.com/sylabs/singularity/releases/download/v${VERSION}/singularity-${VERSION}.tar.gz && \
+    rpmbuild -tb singularity-${VERSION}.tar.gz && \
+    sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/singularity-${VERSION}-1.el7.x86_64.rpm && \
+    rm -rf ~/rpmbuild singularity-${VERSION}*.tar.gz
+```
+
+Alternatively, to build an RPM from the latest master you can 
+[clone the repo as detailed above](#clone-the-repo).  Then create your own
+tarball and use it to install Singularity.  
+
+```
+$ cd $GOPATH/src/github.com/sylabs/singularity && \
+  ./mconfig && \
+  make -C builddir rpm && \
+  sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/singularity-3.0.3-687.gf3da9de.el7.x86_64.rpm # or whatever version you built
```

Comment:
We *could* also do something like this?... Maybe not, just bringing it up.

```suggestion
  sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/singularity-*x86_64.rpm
```

Reply:
Yeah, I wish we could.  But the above command builds 2 versions.  One with debug symbols and one without.  I don't think the wildcard will allow us to distinguish.